### PR TITLE
feat(apps/interface): setup darkmode and lightmode for rainbowkit

### PR DESCRIPTION
**Pull request by math-marcellino**

### CODE REVIEW

**Comment by math-marcellino** [comment]
The test here failed because the coverage does not reach 95%. I was trying to create a test for this `RainbowKitContainer` component, but I kinda confused .-.
cc: @pyk

**Comment by pyk** [comment]
@math-marcellino we need to increase the coverage for some components, let me create separate issue for this

**Comment by math-marcellino** [comment]
> @math-marcellino we need to increase the coverage for some components, let me create separate issue for this

Got it sir, I've converted this PR to `ready for review`. Please kindly check, thank you!

**Comment by pyk** [comment]
@math-marcellino we will merge this once we solve this issues:

- https://linear.app/risedle/issue/RIS-681/choreappsinterface-increase-code-coverage-for-darkmodetoggle

**Comment by pyk** [comment]
@math-marcellino please merge `main` branch to this branch

**Review by pyk — CHANGES_REQUESTED** [review]
@math-marcellino let's move the Wagmi to the RainbowKitContainer too

https://github.com/risedle/monorepo/blob/988cee68f7f0ad5530b5dc7bc3014d515c8358c1/apps/interface/pages/_app.tsx#L23-L67

File: apps/interface/pages/_app.tsx [hunk 1]
```diff
@@ -9,65 +9,12 @@ import "@fontsource/ibm-plex-mono/600.css";
 
 import "@rainbow-me/rainbowkit/styles.css";
```
Comment (pyk) [inline apps/interface/pages/_app.tsx:10]: @math-marcellino We need to move this to `RainbowKitContainer` too

**Review by pyk — CHANGES_REQUESTED** [review]

File: apps/interface/pages/_app.tsx (hunk 1 above)
Comment (math-marcellino) [reply apps/interface/pages/_app.tsx:10]: Ahh yes I forgot about that, lemme fix it now 👍

**Review by pyk — APPROVED** [review]
Thanks @math-marcellino

**Pull request merged**